functions/src/index.ts

```diff
@@ -28,5 +28,6 @@ import * as ReportingSync from './modules/reporting.sync.functions';
 import * as Email from './modules/email.functions';
 import * as SE from './modules/SE.functions';
 import * as Authorization from './modules/Authorization.functions';
+import * as Gmail from './modules/gmail.functions';
```

Comment:
Let's make all modules files either Capital-cased or camel-cased. What is your proposal?

Reply:
So name your file `Gmail.functions.ts`. Other lowercase-modules will be renamed later.

Reply:
File still is not renamed to `Gmail.functions.ts`